### PR TITLE
Fix `create_agenda` method

### PR DESCRIPTION
**Pull request by TomTaehoonKim**
1. Update `create_agenda` using `query_commit_path`
2. Trim long commentent

**Added after last review**
3. Fix comment

### CODE REVIEW

**Review by YoonhoKim0527 — COMMENTED** [review]
LGTM 
But did you assumed 196 to be merged?

**Comment by TomTaehoonKim** [comment]
> LGTM But did you assumed 196 to be merged?

@YoonhoKim0527 yes this pr has dependency to 196.

File: repository/src/lib.rs [hunk 1]
```diff
@@ -248,27 +248,19 @@ impl<T: RawRepository> DistributedRepository<T> {
...
-        // Fetch and convert commits
-        let commits = self.raw.list_ancestors(work_commit, Some(256)).await?;
-        let position = commits
-            .iter()
-            .position(|c| *c == last_header_commit)
-            .expect("TODO: handle the case where it exceeds the limit.");
-
-        // commits starting from the very next one to the last finalized block.
-        let commits = stream::iter(commits.iter().take(position).rev().cloned().map(|c| {
+        // Construct a commit list starting from the next commit of the last finalized block to the `branch_commit`(the most recent commit of the branch)
```
Comment (junha1) [inline repository/src/lib.rs:251]: Please trim this long comment.

Comment (TomTaehoonKim) [reply repository/src/lib.rs]: fixed

**Pull request merged**